src/documents/views.py

```diff
@@ -535,7 +536,7 @@ class DocumentTypeViewSet(ModelViewSet, PermissionsAwareDocumentCountMixin):
             404: None,
         },
     ),
-    email=extend_schema(
+    email_document=extend_schema(
         description="Email the document to one or more recipients as an attachment.",
         request=inline_serializer(
```

Comment:
Ah, one other thing from me, can use the `EmailSerializer` here also? Might be tricky because of the documents thing, but the more structured stuff we give to drf-spectacular the less error-prone the API spec is

Reply:
Hmm, not sure I’d have to play with it to see. Ideally you can use EmailSerializer and just override the documents parameter (or at least document it in the spec), drf-spectacular does have a lot of options but I don’t know off the top of my head

Reply:
See https://github.com/paperless-ngx/paperless-ngx/pull/10666/commits/b234eea472ff732ffd1fdf1664ddb7588bf90503